pgsqltoolsservice/pgtoolsservice_main.py

```diff
@@ -97,7 +97,7 @@ def _create_server(input_stream, output_stream, server_logger):
 
     std_out_wrapped = io.open(sys.stdout.fileno(), 'wb', buffering=0, closefd=False)
 
-    logger.info('PostgreSQL Tools Service is starting up...')
+    logger.info('MySQL Tools Service is starting up...')
```

Comment:
Hardcoding to MySQL ?

Reply:
Right, I made sure to change the string based on the provider in the recent commit.